melodic/lcas-dist.yaml

```diff
@@ -555,10 +555,47 @@ repositories:
     status: developed
   spencer_people_tracking:
     source:
-      test_commits: true
+      test_commits: false
       type: git
-      url: https://github.com/LCAS/spencer_people_tracking.git
-      version: master
+      url: https://github.com/spencer-project/spencer_people_tracking.git
+      version: melodic
+    release:
```

Comment:
This is the problem... As it has not yet been released, the `release` key (and its content) shouldn't be here yet. I suggest @tlind you remove it for now, and then we can release an actual version as soon as the melodic build works. 

Also, I see that you have change the branch to the spencer-project one, not our fork. Generally, not a problem at all.

Reply:
Ok! I wasn't aware that this is a two-stage process. I have removed the entire release block for now.